perma_web/perma/views/service.py

```diff
@@ -88,4 +92,17 @@ def bookmarklet_create(request):
     '''
     tocapture = request.GET.get('url', '')
     add_url = "{}?url={}".format(reverse('create_link'), tocapture)
-    return redirect(add_url)
\ No newline at end of file
+    return redirect(add_url)
+
+@login_required
+@user_passes_test(lambda user: user.is_staff or user.is_registrar_user())
```

Comment:
Tiniest of suggestions: can we use perma.utils.user_passes_test_or_403 here instead? You can get into some funny redirect situations with the plain user_passes_test decorator.